mathcomp/Makefile.coq.local

```diff
@@ -0,0 +1,17 @@
+OUTPUT_TESTS=test_suite/output.v
+
+OUTPUT_ARTIFACTS=$(OUTPUT_TESTS:%.v=%.v.out.new)
+
+post-all:: $(OUTPUT_ARTIFACTS)
+	@for f in $(OUTPUT_TESTS); do\
+	  diff -u $$f.out $$f.out.new > /dev/null;\
+	  if [ $$? -ne 0 ]; \
+	    then echo "Output of file $$f differs!";\
+		diff -u $$f.out $$f.out.new;\
+		exit 1;\
+	  fi;\
+	done
+
+$(OUTPUT_ARTIFACTS): %.v.out.new: %.v all/all.vo
+	$(COQC) $(COQFLAGS) $(COQLIBS) $< > $<.out.new
+	
```

Comment:
This should be moved to `Makefile.test-suite.local`. Currently, the use of `coq_makefile` is split into two parts: the library and test-suite, because generating `test_suite/hierarchy_test.v` requires `all/all.vo`.

Reply:
@gares do you confirm that this PR should really be in release 1.11?